src/main/java/mate/academy/spring/controller/MovieSessionController.java

```diff
@@ -0,0 +1,66 @@
+package mate.academy.spring.controller;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.dto.request.MovieSessionRequestDto;
+import mate.academy.spring.dto.response.MovieSessionResponseDto;
+import mate.academy.spring.model.MovieSession;
+import mate.academy.spring.service.MovieSessionService;
+import mate.academy.spring.service.mapper.MovieSessionMapper;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/movie-sessions")
+public class MovieSessionController {
+    private final MovieSessionService movieSessionService;
+    private final MovieSessionMapper movieSessionMapper;
+
+    public MovieSessionController(MovieSessionService movieSessionService,
+                                  MovieSessionMapper movieSessionMapper) {
+        this.movieSessionService = movieSessionService;
+        this.movieSessionMapper = movieSessionMapper;
+    }
+
+    @PostMapping
+    public MovieSessionResponseDto create(@RequestBody MovieSessionRequestDto movieSession) {
+        return movieSessionMapper.doDto(movieSessionService
+                .add(movieSessionMapper.toModel(movieSession)));
+    }
+
+    @GetMapping("/available")
+    public List<MovieSessionResponseDto> getAvailableMovieSession(@RequestParam Long movieId,
+                                                                  @RequestParam
+                                                                  @DateTimeFormat
+                                                                          (pattern = "dd.MM.yyyy")
+                                                                  LocalDate date) {

```

Comment:
```suggestion
    public List<MovieSessionResponseDto> getAvailableMovieSession(
    @RequestParam Long movieId,
    @RequestParam @DateTimeFormat(pattern = "dd.MM.yyyy") LocalDate date
    ) {
```
you can pass parameters in this way